examples/incremental_learning/dog_croissants_classification/training/mobilenet_v2.py

```diff
@@ -0,0 +1,34 @@
+import mindspore as ms
+from mindvision.classification.models import mobilenet_v2
+from mindvision.dataset import DownLoad
+
+
+
+class mobilenet_v2_fine_tune:
+    # TODO: save model
+    def __init__(self, base_model_url=None):
+        models_download_url = "https://download.mindspore.cn/vision/classification/mobilenet_v2_1.0_224.ckpt"
```

Comment:
Would models_download_url be better as a configurable parameter?

Reply:
I have removed this download part. It's unnecessary because users should prepare `ckpt` file by themselves when they prepare to run this demo. I have provided `curl` command in README